modules/plugin-search/plugin-search.css

```diff
@@ -1,4 +1,27 @@
+.plugin-card-jetpack-plugin-search h3 {
+	margin: 0 0 4px 0;
+}
+
+.plugin-card-jetpack-plugin-search .column-name,
+.plugin-card-jetpack-plugin-search .column-description {
+	margin-right: 20px;
+}
+
+.plugin-card-jetpack-plugin-search .action-links {
+	overflow: auto;
+	position: static;
+}
+
+@media screen and (max-width: 1100px) and (min-width: 782px), (max-width: 480px) {
+	.plugin-card-jetpack-plugin-search .action-links {
+		margin-left: 0;
+	}
+}
+
 .plugin-card-jetpack-plugin-search .plugin-action-buttons {
+	margin: 0;
+	width: 100%;
+	text-align: left;
 	white-space: nowrap;
 }
 
```

Comment:
Can you add the following in here:
```
@media screen and (max-width: 1100px) and (min-width: 782px), (max-width: 480px)
.plugin-card-jetpack-plugin-search .plugin-action-buttons li {
    display: block;
}
```

It's untested, but I think will fix a display bug at tablet resolutions.

Reply:
Added